utils/spaceship.asyncState/index.js

```diff
@@ -0,0 +1,23 @@
+export const PENDING = 'pending';
+export const SUCCESS = 'success';
+export const FAILURE = 'failure';
+
+export default asyncState = stateName => {
+  switch (stateName) {
+    case PENDING:
+    case SUCCESS:
+    case FAILURE:
+      return {
+        pending: false,
+        success: false,
+        failure: false,
+        [stateName]: true
+      };
+    default:
+      return {
+        pending: false,
+        success: false,
+        failure: false
+      };
+  }
+};
```

Comment:
End Of Line Plz At all file you write.

Reply:
I can't understand well... what `asyncState` function does. 

Could you explain this logic?

Reply:
thanks 🙌
I applied convention this commit 77307c1

Reply:
I add explain to #9 🙌